modules/olm-policy-scoping-operator-install.adoc

```diff
@@ -10,6 +10,11 @@ To provide scoping rules to Operator installations and upgrades on Operator Life
 
 Using this example, a cluster administrator can confine a set of Operators to a designated namespace.
 
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
```

Comment:
dedicated-admin role?

Reply:
This actually isn't included in the OSD/ROSA docs. I just added the prereqs for OCP since they were missing.